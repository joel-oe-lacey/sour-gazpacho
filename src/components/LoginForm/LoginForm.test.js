//need to test local state changing 
//validate user should hit fetch
//need to mock out fetch
//need to jest fn preventDefault

//need a snapshot test
//test mapDispatchToProps

describe('LoginForm', () => {
    describe('LoginForm Container', () => {
        //unit tests we already know how to write
    });

    describe('mapDispatchToProps', () => {
        //unit tests we already know how to write
    });

})