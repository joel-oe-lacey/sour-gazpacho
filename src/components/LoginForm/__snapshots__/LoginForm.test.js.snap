// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm LoginForm Container should match snapshot 1`] = `
<section
  className="form-container"
>
  <form
    className="form"
  >
    <h1>
      Sour Gazpacho
    </h1>
    <input
      name="email"
      onChange={[Function]}
      placeholder="Email..."
      required={true}
      type="text"
    />
    <input
      name="password"
      onChange={[Function]}
      placeholder="Password..."
      required={true}
      type="text"
    />
    <h4 />
    <button
      onClick={[Function]}
    >
      Login
    </button>
  </form>
</section>
`;
